pkg/proxy/accesslog/log.go

```diff
@@ -17,10 +17,19 @@ package accesslog
 import (
 	"bufio"
 	"encoding/json"
+	"net"
+	"net/http"
+	"net/url"
 	"os"
+	"strconv"
+	"time"
 
+	"github.com/cilium/cilium/common/addressing"
```

Comment:
Package github.com/cilium/cilium/pkg/proxy/accesslog is a public API. It's used by third-party software. Adding more transitive dependencies should be avoided. Please remove those additional dependencies. If that means moving the code depending on this out of github.com/cilium/cilium/pkg/proxy/accesslog, do that.

Reply:
OK, was not aware of this. OK to remove the http.Request from the log record, though?